sdk/core/System.ClientModel/src/ModelReaderWriter/IJsonModel.cs

```diff
@@ -3,33 +3,32 @@
 
 using System.Text.Json;
 
-namespace System.ClientModel.Primitives
+namespace System.ClientModel.Primitives;
```

Comment:
These changes just move the model files to use [file scoped namespaces](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces) as we do elsewhere in ClientModel.

Reply:
Is it worth adding some project level editorconfig settings to enforce this?

Reply:
I like that idea ... are there examples in our repo I could copy - do you know offhand?

Reply:
I don't see any examples, but you define a new editorconfig file in System.ClientModel directory containing the following contents:

```
[*.cs]
csharp_style_namespace_declarations = file_scoped:error
```

Reply:
FWIW I noticed that none of the other defaults we are setting in the root EditorConfig are being applied because they all filter on Azure.* as the project directory - https://github.com/Azure/azure-sdk-for-net/blob/main/.editorconfig#L18

Reply:
Filed https://github.com/Azure/azure-sdk-for-net/issues/41430, but let me see how hard it is to add System.* to the existing one in this PR...

Reply:
And TY!  -- addressed, assuming I don't break CIs...